docs/_posts/examples/3400-01-28-road-profile-extrusion.html

```diff
@@ -0,0 +1,55 @@
+--- 
+layout: example 
+category: example 
+title: Extrude a 3D profile.
+description: Create a 3D road or activity profile using fill-extrude-height.
+tags: 
+  - layers 
+---
+
+<div id='map'>
+</div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/dark-v9',
+    center: [-122.4662, 37.7989],
+    zoom: 14,
+    pitch: 50,
+    bearing: 0
+});
+
+map.on('load', function() {
+
+    // source GeoJSON data for visual is at
+    // https://gist.github.com/ryanbaumann/f196a84c4c5728d860db0fc7a19d0c9b
+    map.addSource("roadProfile", {
+        'type': 'vector',
+        'url': 'mapbox://examples.ciu640p5o0ifh2pjwzwd6ga98-0h1qy'
+    });
```

Comment:
@lucaswoj 
* I updated the data to source from geojson url hosted on the /assets folder of the doc.
* I added comments to the DDS functions
* I linked to a gist explaining how to get data in this format for creating the example.